triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -723,6 +735,22 @@ private[lf] class Runner private (
         ctx.copy(value = converter.fromHeartbeat)
     }
 
+  private[this] def conflateMsgValue: TriggerContextualFlow[SValue, SValue, NotUsed] = {
```

Comment:
if I understand this correctly this looks like your list can potentially get arbitrarily large. Have you considered putting a bound on this?

Reply:
if you make the bound configurable, people can even recover the existing behavior which may be useful if there are edge cases we have not anticipated.

Reply:
Will replace use of `conflateWithSeed` with `batch`. This will batch elements until a configurable max size and then it will back pressure.